.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: MPF CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        project: [MPF, MPF.Check]
```

Comment:
At a later point, it may be worthwhile to make separate GHA workflow files to avoid having to do too many excludes.